distributed/utils_test.py

```diff
@@ -1125,6 +1125,8 @@ def _terminate_process(proc: subprocess.Popen, terminate_timeout: float) -> None
             proc.send_signal(signal.SIGINT)
         try:
             proc.communicate(timeout=terminate_timeout)
+        except subprocess.TimeoutExpired:
+            print("Process did not terminate after SIGINT; sending SIGKILL")
```

Comment:
I think I'm OK if the process does not shut down with just SIGINT so I'm catching this and just print the information

Reply:
Thought about this again. I think we should raise after all. Having this not work well is a little unhealthy